essentials-module-utilities/src/main/kotlin/io/github/nyliummc/essentials/commands/FlightCommand.kt

```diff
@@ -26,30 +26,27 @@ package io.github.nyliummc.essentials.commands
 
 import com.mojang.brigadier.CommandDispatcher
 import com.mojang.brigadier.context.CommandContext
-import io.github.ladysnake.pal.AbilitySource
 import io.github.ladysnake.pal.Pal
 import io.github.ladysnake.pal.VanillaAbilities
 import io.github.nyliummc.essentials.api.builders.Command
-import io.github.nyliummc.essentials.api.builders.Text
 import net.minecraft.command.arguments.EntityArgumentType
 import net.minecraft.server.command.ServerCommandSource
 import net.minecraft.server.network.ServerPlayerEntity
 import net.minecraft.text.LiteralText
 
 object FlightCommand {
-
-    private val abilitySource = Pal.getAbilitySource("essentials", "flight");
+    val ESSENTIALS_ABILITY_FLY = Pal.getAbilitySource("essentials", "flight");
 
     fun register(dispatcher: CommandDispatcher<ServerCommandSource>) {
         Command.builder(dispatcher) {
             command("flight", "fly") {
                 requires { it.hasPermissionLevel(4) }
 
-                executes (::toggleFlightSelf)
+                executes (FlightCommand::toggleFlightSelf)
```

Comment:
Why did you change that?

Reply:
not quite sure why that happened, will fix